File: test/bench/fib/fib.js
function fib(n) {
    if (n < 2) {
        return 1;
    } else {
        return fib(n - 2) + fib(n - 1);
    }
}

var before = (new Date()).getTime();
console.log(fib(20));
var after = (new Date()).getTime();
console.log(after - before);

File: test/bench/fib/fib.rb
def fib(n)
  if n == 0
    1
  elsif n == 1
    1
  else
    fib(n - 2) + fib(n - 1)
  end
end

before = Time.now
puts fib(25)
after = Time.now
puts (after - before)
